examples/BLUEPRINT/base/base_tutorial.ipynb

```diff
@@ -49,7 +46,7 @@
         "    default_params = [\n",
         "        [\"n_TF\", \"Number of TF coils\", 16, \"N/A\", None, \"Input\"],\n",
         "        [\"coolant\", \"Coolant\", \"Water\", None, \"Divertor coolant type\", \"Common sense\"],\n",
-        "        [\"T_in\", \"Coolant inlet T\", 80.0, \"°C\", \"Coolant inlet T\", None],\n",
+        "        [\"T_in\", \"Coolant inlet T\", 80.0, \"\u00b0C\", \"Coolant inlet T\", None],\n",
```

Comment:
Any reason for this change? (I guess it relates to the on-going UTF-8 vs ascii issues).

Reply:
I think possibly @CoronelBuendia changed it in the py file? I remember seeing it and assuming the same...

Reply:
Ah, fair enough, makes sense that the notebook aligns with the .py then.